src/test/custom_nodes/node_perform_different_operations.cpp

```diff
@@ -68,11 +68,12 @@ int execute(const struct CustomNodeTensor* inputs, int inputsLength, struct Cust
     }
 
     // prepare outputs
-    *outputsLength = 1;
+    *outputsLength = 2;
     *outputs = (struct CustomNodeTensor*)malloc(*outputsLength * sizeof(CustomNodeTensor));
-    float* result = (float*)malloc(OPS_END * valuesPerTensor * sizeof(float));  // dummy input size * numbr of ops
+    float* result = (float*)malloc(OPS_END * valuesPerTensor * sizeof(float));  // dummy input size * number of ops
+    float* resultFactors = (float*)malloc(OPS_END * OPS_END * sizeof(float));   // dummy input size * number of ops
 
-    CustomNodeTensor& resultTensor = *outputs[*outputsLength - 1];
+    CustomNodeTensor& resultTensor = (*outputs)[0];
```

Comment:
You have used `0` as index and then `*outputsLength-1`. Stick to one 0/1 or -1/-2